README.md

```diff
@@ -20,10 +20,12 @@
 
 This is the JS SDK for the NEO blockchain platform. This project aims to be a lightweight library focused on providing blockchain interactions in the browser.
 
-It is currently in use by [Neon](https://github.com/CityOfZion/neon-wallet/).
+Neon-JS is used internally by [Neon Wallet](https://github.com/CityOfZion/neon-wallet/) and many other libraries and applications.
 
 Visit the [docs](https://dojo.coz.io/neo3/neon-js/index.html) to learn how to use this library!
 
+> Although Neon-JS offers a comprehensive set of features, [NeonDappkit](https://github.com/CityOfZion/neon-dappkit) and [WalletConnectSDK](https://github.com/CityOfZion/wallet-connect-sdk) are specifically tailored for `Dapp development`, providing a more user-friendly experience for this particular purpose.
```

Comment:
Opinion: I think a more active voice would work well here. Just start the sentence with `For Dapp development, take a look at NeonDappkit... `

Alternatively, restructure line23 to a list:

```
Neon-JS is used by:

- Neon Wallet, a GUI wallet application for NEO
- NeonDappkit, a framework for building Dapps on NEO
- WalletConnectSDK
```

Reply:
I've changed the first text, but I would like to avoid the line23 change to keep it short.